code/data_quality_tab/data_monitoring_completeness.R

```diff
@@ -163,24 +149,23 @@ smr_completeness_2 <- smr_completeness %>%
   left_join(change_df)%>%
   left_join(flag_df)
 
-#finally, we store a record of the earliest month and latest month the bar charts were produced from
+#store record of the earliest month and latest month the bar charts were produced from
 #this will be displayed in the text above the table when the app is run
-date1 <- smr_completeness %>% 
-  filter(event_year == min(event_year) & event_month == min(event_month))%>%
-  distinct(event_year, event_month, month_name) %>% 
-  rename(year_1=event_year, month_1=event_month, month_name_1=month_name)
 
-date2 <- smr_completeness %>% 
-  filter(event_year == max(event_year) & event_month == max(event_month))%>%
-  distinct(event_year, event_month, month_name) %>% 
-  rename(year_2=event_year, month_2=event_month, month_name_2=month_name)
+min_date <- min(smr_completeness$event_month)
+max_date <- max(smr_completeness$event_month)
 
-comp_barchart_dates <- cbind(date1, date2)
+comp_barchart_dates <- data.frame(min_year = year(min_date), 
+        min_month= month(min_date, label = TRUE, abbr = FALSE),
+        max_year=year(max_date),
+        max_month=month(max_date, label = TRUE, abbr = FALSE))
 
 # Write the Outputs --------------------------------------------------------
 
 #write out the table output so that it can be imported in global.R
-write_csv(smr_completeness_2, here::here("data", "smr_completeness.csv"))
+write_csv(smr_completeness_2, 
+          "/conf/Data_Quality_Dashboard/data/smr_completeness.csv")
```

Comment:
This will be affected by the decision around data storage and the use of relative file paths.